core/src/main/java/org/apache/seata/core/rpc/netty/v0/ProtocolDecoderV0.java

```diff
@@ -144,22 +144,15 @@ public RpcMessage decodeFrame(ByteBuf in) {
 
     @Override
     protected Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception {
-        Object decoded;
         try {
-            decoded = super.decode(ctx, in);
-            if (decoded instanceof ByteBuf) {
-                ByteBuf frame = (ByteBuf)decoded;
-                try {
-                    return decodeFrame(frame);
-                } finally {
-                    frame.release();
-                }
+            if (in instanceof ByteBuf) {
+                return decodeFrame(in);
             }
         } catch (Exception exx) {
             LOGGER.error("Decode frame error, cause: {}", exx.getMessage());
             throw new DecodeException(exx);
         }
-        return decoded;
+        return in;
```

Comment:
why return in？

Reply:
Running here means that an exception or incorrect type was encountered. 
Follow the original logic, skip this processing, return **in**